### PR TITLE
Add information about ignoreTmpls property

### PR DESCRIPTION
**Pull request by serhiyzhovnir**
## Purpose of this pull request

This pull request (PR) adds information about `ignoreTmpls` property.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/ui_comp_guide/concepts/ui_comp_template_literals.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/js/lib/core/class.js#L86-L89
-  https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/js/form/element/abstract.js#L60-L62

whatsnew 
Added a `ignoreTmpls property' section to [Template Literals in UI Components](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/concepts/ui_comp_template_literals.html) page.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/ui_comp_guide/concepts/ui_comp_template_literals.md [hunk 1]
```diff
@@ -40,6 +40,23 @@ As a result, a template literal used in the value of one the objects listed abov
...
+### The `ignoreTmpls` property
+
+The `ignoreTmpls` property is an object that allows preventing template processing for selected properties.
+
+```javascript
+return Element.extend({
+    defaults: {
+        value: 'some component value',
+        property: '${ $.value }', // equal to 'some component value' after component initialization
+        propertyIgnoreTempls: '${ $.value }', // equal to '${ $.value }' after component initialization
```
Comment (rogyar) [inline src/guides/v2.3/ui_comp_guide/concepts/ui_comp_template_literals.md]: This part might be confusing. We may either remove `property` or add an extra comment for `propertyIgnoreTempls` that  `propertyIgnoreTempls` is the property for which template literals will be ignored. Also, we may add an extra comment to `ignoreTmpls` that this section contains list of properties with template literals ignored.

**Comment by serhiyzhovnir** [comment]
Hi @rogyar 
The PR was adjusted.
Could you, please, check it again.
Thank you!

File: src/guides/v2.3/ui_comp_guide/concepts/ui_comp_template_literals.md (hunk 1 above)
Comment (rogyar) [reply src/guides/v2.3/ui_comp_guide/concepts/ui_comp_template_literals.md]: Great. Thank you!

**Comment by dobooth** [comment]
running tests

**Pull request merged**